### DISCUSSION TITLE
Feature Request: Implement separate per-panel variables to ad-hoc filtering only needed panel (Table, Logs)

### DISCUSSION THREAD

**Discussion by MurzNN** [grafana/grafana#38297] [Dashboards, Panels & Visualizations]
**What would you like to be added**:
Add per-panel variables, that allow to ad-hoc filtering of one separate panel (Table, Logs) by variable value, not the whole dashboard.

**Why is this needed**:
We already have per-dashboard Variables, that shows at top of dashboard page. But sometimes we need to filter only one panel at dashboard, especially the Table and Logs visualizations. 

This feature will be very useful for quick filtering the long log lines panels to find needed value, without touching other panels in dashboard (and reload their data).

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by AbelThorne** [comment]
Yes, please! I've been looking for this for a long time also :-)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!

**Comment by MurzNN** [comment]
Opened an issue instead: https://github.com/grafana/grafana/issues/82496
